### PR TITLE
[devtools-blog] What's New in DevTools (Chrome 92)

### PR DESCRIPTION
**Pull request by chybie**

### CODE REVIEW

**Comment by chybie** [comment]
@robdodson similar error, there's not gitignore, causing the build failed, not related to the changes in this PR.

**Review by hashseed — APPROVED** [review]

**Comment by robdodson** [comment]
Broken build should be fixed now. If you rebase against `main` the tests should pass.

File: site/en/blog/new-in-devtools-92/index.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+{% endAside %}
+
+
+## CSS grid editor {: #grid-editor }
+A highly requested feature. You can now preview and author CSS Grid with the new CSS Grid editor! 
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/mV9Ac7QAD8vVPoiqmii6.png", alt="CSS Grid editor", width="800", height="486" %}
+
+When an HTML element on your page has `display: grid` or `display: grid` applied to it, you can see an icon appear next to it in the Styles pane. Click the icon to toggle the CSS grid editor. Here you can preview the potential changes with the on screen icons (e.g. `justify-content: space-around`) and author the grid appearance with just one click.
```
Comment (mathiasbynens) [inline site/en/blog/new-in-devtools-92/index.md:29]: This seems like a typo:

> `display: grid` or `display: grid`

Comment (chybie) [reply site/en/blog/new-in-devtools-92/index.md:29]: Fix #881. Please help to review.

**Pull request merged**